dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -400,7 +400,7 @@ export const renderElement = ({
 			return (
 				// Deferring interactives until CPU idle achieves the lowest Cumulative Layout Shift (CLS)
 				// For more information on the experiment we ran see: https://github.com/guardian/dotcom-rendering/pull/4942
-				<Island defer={{ until: 'idle' }}>
+				<Island priority="critical" defer={{ until: 'visible' }}>
```

Comment:
We made this change because you cannot defer until `idle` for a `critical` priority – we should probably have brought more attention to this change to @guardian/commercial-dev

Reply:
After a catchup with @mxdvl regarding this change it turns out it is safe to use `idle` in priority `critical` because there is a timeout in `whenIdle` function. There is a new change introduced here https://github.com/guardian/dotcom-rendering/pull/11077